File: URI/1962/java/Main.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);

		Integer n = input.nextInt();
		for (int i = 0; i < n; i++) {
			Integer year = input.nextInt();
			if (year < 2015) {
				Integer diff = 2015 - year;
				System.out.println(diff.toString() + " D.C.");
			}
			else {
				Integer diff = year - 2014;
				System.out.println(diff.toString() + " A.C.");
			}
		}
		input.close();
	}
}

File: URI/1962/cpp/main.cpp
#include <stdio.h>
#include <iostream>

using namespace std;

int main() {
	int n;
	cin >> n;

	int year, diff;
	for (int i = 0; i < n; i++){
		cin >> year;
		if (year < 2015) {
			diff = 2015 - year;
			cout << diff << " D.C.\n";
		}
		else {
			diff = year - 2014;
			cout << diff << " A.C.\n";
		}
	};

	return 0;
}
